test/functional/test_framework/test_framework.py

```diff
@@ -260,7 +291,12 @@ def set_binary_paths(self):
                 "src",
                 binary + self.config["environment"]["EXEEXT"],
             )
-            setattr(self.options, attribute_name, os.getenv(env_variable_name, default=default_filename))
+            setattr(paths, attribute_name, os.getenv(env_variable_name, default=default_filename))
+
+        # BITCOIN_CMD environment variable can be specified to invoke bitcoin
+        # binary wrapper binary instead of other executables.
```

Comment:
```suggestion
        # BITCOIN_CMD environment variable can be specified to invoke bitcoin
        # wrapper binary instead of other executables.
```